content/dstu2/care-plan.md

```diff
@@ -23,6 +23,8 @@ _Implementation Notes_
 
 * The description of the CarePlan is included in the [CarePlan.text.div] element as escaped XHTML. We will be making changes to increase the usability of the description.
 
+* In DSTU2 CareTeam is not a stand alone resource. It is defined by the [Argonaut profile] as a CarePlan with participants. A CareTeam will be returned only if a CareTeam specific category code is supplied.
```

Comment:
Since we're not specifically returning a care team, we might re-phrase this a bit.

>  A CarePlan with care team participants will be returned only if a care team specific category code is supplied.